### PR TITLE
TST smoke tests for bugfixes for issues 409 and 410

### PR DESCRIPTION
**Pull request by eickenberg**
Adds test cases that run the scenarios where #410 and #409 ran into bugs.

### CODE REVIEW

File: nilearn/plotting/tests/test_img_plotting.py [hunk 1]
```diff
@@ -76,7 +77,11 @@ def test_plot_functions():
     for func in [plot_anat, plot_img, plot_stat_map,
                  plot_epi, plot_glass_brain,
                  partial(plot_stat_map, symmetric_cbar=True),
-                 partial(plot_stat_map, symmetric_cbar=False)]:
+                 partial(plot_stat_map, symmetric_cbar=False),
+                 partial(plot_stat_map, symmetric_cbar=False, vmax=10),
+                 partial(plot_stat_map, symmetric_cbar=True, vmax=10),
+                 partial(plot_stat_map, colorbar=False)
```
Comment (lesteve) [inline nilearn/plotting/tests/test_img_plotting.py]: Should we try to add similar tests for plot_stat_map where the data has both positive and negative values?

**Comment by GaelVaroquaux** [comment]
Can we make the array a bit smaller: right now it is 20x20x20. I am a bit afraid at the cumulated time that this test is taking.

**Comment by eickenberg** [comment]
Gladly, but I cannot go lower than 7 on axes x and y because of the axis slicer who seems to expect at least this. So now I am running all the previous tests for positive, negative and mixed data, that means I am doing 3 * 10 = 30 plots. In total this takes around 55 seconds.

The only way to reduce this significantly is by doing less test cases, not by reducing the image size - we are almost in pure plotting overhead here. I could do positive/negative/mixed only for plot_stat_map for instance.

**Comment by lesteve** [comment]
There is an issue for speeding up the plotting tests #408. I'd be in favour of merging this and tackling the plotting tests slowness after the release.

**Comment by AlexandreAbraham** [comment]
If plotting is the bottleneck, you can use the backend `template` (I have seen that currently the tests use `svg`). `template` is an empty backend that doesn't draw anything. But I don' t think that the speed up will be significant.

**Comment by lesteve** [comment]
OK merging this one. @AlexandreAbraham can you add your suggestion of using the `template` bottleneck in #408 so that we don't forget about it?

**Comment by GaelVaroquaux** [comment]
55s is really nasty. We'll have to reduce this. We are accumulating
technical debt here.
